com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariableSerialization.cs

```diff
@@ -28,6 +28,40 @@ internal interface INetworkVariableSerializer<T>
         public void Duplicate(in T value, ref T duplicatedValue);
     }
 
+    /// <summary>
+    /// Packing serializer for bytes
+    /// </summary>
+    internal class ByteSerializer : INetworkVariableSerializer<byte>
```

Comment:
Adding a specific serializer class like this is unnecessary for bytes. `byte` can just use `UnmanagedTypeSerializer<byte>`. There is no packing implementation for byte because the packer cannot reduce anything below 1 byte anyway, so when you call `WriteValueBitPacked` on a byte, the result is that the byte gets converted to a short, which then gets packed as a short, which means for any value where the top bit of the byte is set, the encoding method will result in this byte taking up two bytes in the buffer instead of one, and for other values you get no savings since one byte is the smallest transfer unit the system supports.